pkg/task/backup.go

```diff
@@ -148,12 +148,18 @@ func RunBackup(c context.Context, cmdName string, cfg *BackupConfig) error {
 		return err
 	}
 
-	valid, err := client.FastChecksum()
-	if err != nil {
-		return err
-	}
-	if !valid {
-		log.Error("backup FastChecksum mismatch!")
+	if cfg.LastBackupTS == 0 {
+		valid, err := client.FastChecksum()
+		if err != nil {
+			return err
+		}
+		if !valid {
+			log.Error("backup FastChecksum mismatch!")
+			return errors.Errorf("mismatched checksum")
+		}
+
+	} else {
+		log.Warn("Skip fast checksum in incremental backup")
```

Comment:
Could you add some comments/TODO about why fast checksum should be skipped?
Also, could you change it to `Info`?